metadata.json

```diff
@@ -13,6 +13,7 @@
     {"name":"puppetlabs/transition","version_requirement":">= 0.1.0 < 2.0.0"},
     {"name":"herculesteam/augeasproviders_core","version_requirement":">= 2.1.5 < 4.0.0"},
     {"name":"herculesteam/augeasproviders_shellvar","version_requirement":">= 1.2.0 < 5.0.0"}
```

Comment:
Fixed.

Reply:
> Can you also please fix the tests?

I'm afraid I don't know how to make rspec tests.  Would anything really need to be changed as all I've done is ensure that the parent directories of the export get created.

Reply:
@derdanne, the tests have been broken for a while -- at least since I created a fork.  I do not know what to do to fix them.  Can you advise just a little bit to get me started?